contributor-docs/release-guide.md

```diff
@@ -17,44 +14,49 @@ limitations under the License.
 
 # Apache Beam Release Guide
 
-{{< toc >}}
```

Comment:
I actually think that losing this is a big downside. Could we add one that manually links to the sections? Obviously not as ideal as one that automatically stays up to date, but still probably very helpful

Reply:
It can be added automatically, but it is automatically generated by GitHub: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

Do you also want it inline?

Reply:
Demo via https://github.com/apache/beam/blob/2af1a367b6797819be77391a3173797d6da4ab03/contributor-docs/release-guide.md

![image](https://github.com/apache/beam/assets/575353/1369c4f5-8433-4844-b50d-c423fd0ac522)

![image](https://github.com/apache/beam/assets/575353/93152316-e4e1-4ac4-9fd2-fcf8392f545d)

Reply:
Oh interesting, I have never noticed this feature. I expect others haven't either, but its probably fine? Up to you I guess.

Reply:
BTW I somehow thought there was a `[TOC]` thing we could add here to get both, which I'd be fine with, but I couldn't find it.

Reply:
AFAIK there isn't, but I also didn't know about the other version of TOC that you pointed out.